### PR TITLE
feat(icons): added `droplet-off` icon

### PR DESCRIPTION
**Pull request by Footagesus**
## What is the purpose of this pull request?
- [x] New Icon

### Description
Added new `droplet-off` icon.

### Icon use case

Water shut-off, humidity control, fluid management, leak control, filtration, water flow monitoring, environmental systems, irrigation systems, household appliances, condensation prevention.

### Concept

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] The icons are solely my own creation.
- [x] The icons were originally created in #<issueNumber> by @Footagesus
- [x] I've based them on the following Lucide icons: `saturn`
- [ ] I've based them on the following design: 

### Naming

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting
- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Review by ericfennis — APPROVED** [review]
LGTM

File: icons/droplet-off.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Footagesus"
```
Comment (jguddas) [inline icons/droplet-off.json:4]: Should inherit the contributors.

**Review by jguddas — CHANGES_REQUESTED** [review]
The icon has also shifted, we should fix that.

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M18.84+13.36C18.5+12.2+18+11+16+9.5c-2-1.6-3.5-4-4-6.5+0+.6-.8+2.4-.82+2.46%22+/%3E%3Cpath+d=%22M22+22+2+2%22+/%3E%3Cpath+d=%22M8.8+8.8c-3+2.3-3.82+4.4-3.8+6.2a7+7+0+0+0+13.18+3.18%22+/%3E">icons<br/>Open lucide studio</a>

**Comment by jguddas** [comment]
Here is the autogenerated one from a long time ago, please don't add me as a contributor, for this.

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M18.715+13.186C18.29+11.858+17.384+10.607+16+9.5c-2-1.6-3.5-4-4-6.5a10.7+10.7+0+0+1-.884+2.586%22+/%3E%3Cpath+d=%22m2+2+20+20%22+/%3E%3Cpath+d=%22M8.795+8.797A11+11+0+0+1+8+9.5C6+11.1+5+13+5+15a7+7+0+0+0+13.222+3.208%22+/%3E&name=droplet&base=droplet">icons<br/>Open lucide studio</a>

**Comment by Footagesus** [comment]
🤔🤔

**Pull request closed without merging**